CHANGELOG/CHANGELOG-1.13.md

```diff
@@ -0,0 +1,90 @@
+# Changelog 1.13
+
+## 1.13.0 - 2023-07-24
+
+### Added
+
+- Add AdminNetworkPolicy support in Antrea to align with K8s NetworkPolicy API, and document the introduction and usage. ([#5170](https://github.com/antrea-io/antrea/pull/5170) [#5270](https://github.com/antrea-io/antrea/pull/5270), [@Dyanngg])
+- Support DSR mode for Service's external addresses including LoadBalancerIPs and ExternalIPs on AntreaProxy. ([#5202](https://github.com/antrea-io/antrea/pull/5202) [#5251](https://github.com/antrea-io/antrea/pull/5251), [@tnqn])
+- Containerize Windows userspace OVS processes and run them in a container of the Antrea Agent Pod to align with the Linux design. ([#4936](https://github.com/antrea-io/antrea/pull/4936) [#5052](https://github.com/antrea-io/antrea/pull/5052), [@rajnkamr] [@Atish-iaf])
+- Add a new option `ContainerRuntime` to allow user to configure the container runtime while using the script `PrepareNode.ps1` on K8s Windows Node. ([#5071](https://github.com/antrea-io/antrea/pull/5071), [@NamanAg30])
+- Add support of TLS, HTTP, and HTTPs protocols for FlowAggregator to connect to the ClickHouse DB, and allow users to specify the CA certificate for TLS and HTTPs. ([#5171](https://github.com/antrea-io/antrea/pull/5171), [@yuntanghsu])
+- Enhance Antrea L7 NetworkPolicy to support the TLS protocol. ([#4932](https://github.com/antrea-io/antrea/pull/4932), [@hongliangl])
+- Add command `antctl upgrade api-storage` to support resource storage version migration for Antrea CRDs". ([#5198](https://github.com/antrea-io/antrea/pull/5198), [@hongliangl])
+- Add Egress information to flow records for Pod-to-external flows in FlowExporter. ([#5088](https://github.com/antrea-io/antrea/pull/5088), [@dreamtalen])
+- Increase accuracy of Pod information in the flow records by adding a Pod store in FlowExporter and FlowAggregator for them to fetch the Pod information. ([#5185](https://github.com/antrea-io/antrea/pull/5185), [@yuntanghsu])
+- Add support for Service annotation `service.kubernetes.io/topology-mode` in AntreaProxy since the old `service.kubernetes.io/topology-aware-hints` annotation has been deprecated in Kubernetes 1.27. ([#5241](https://github.com/antrea-io/antrea/pull/5241), [@mengdie-song])
+- Support the `service.kubernetes.io/service-proxy-name` label in AntreaProxy to align with KEP 2447. ([#4973](https://github.com/antrea-io/antrea/pull/4973), [@hongliangl])
+- Add a new Prometheus metric to represent the number of packets dropped by OVS meter. ([#5165](https://github.com/antrea-io/antrea/pull/5165), [@mengdie-song])
+- Add support for the `sort-by` flag in more `antctl get` commands for more fields. ([#4346](https://github.com/antrea-io/antrea/pull/4346), [@jainpulkit22])
+- Add the `kubeAPIServerOverride` option to allow user to override the kube-apiserver address for antrea-controller. ([#5056](https://github.com/antrea-io/antrea/pull/5056), [@tnqn])
```

Comment:
```suggestion
- Add the `kubeAPIServerOverride` option to allow user to override the kube-apiserver Endpoint for antrea-controller. ([#5056](https://github.com/antrea-io/antrea/pull/5056), [@tnqn])
```

Reply:
I feel `address` is a more common way for the kube-apiserver, I am not sure what the Endpoint is referring to? @tnqn  any comment?

Reply:
Yes, address looks more proper to me. The capitalized Endpoint might be misleading.